tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartGrammaticalForms.java

```diff
@@ -400,8 +400,13 @@ public void writeSubcharts(Anchors anchors) throws IOException {
                     String quantity = shortUnit.contentEquals("generic") ? "temperature" : uc.getQuantityFromUnit(shortUnit, false);
 
                     String gender = UnitPathType.gender.getTrans(cldrFile, "long", shortUnit, null, null, null, null);
+                    gender = gender == null ? "n/a" : gender;
                     Set<String> systems = uc.getSystems(shortUnit);
 
+                    if (unitCell == null || quantity == null || gender == null || sizeInBaseUnits.value == null) {
+                        throw new IllegalArgumentException("No best base unit for: " + shortUnit);
+                    }
+
```

Comment:
`gender == null` can't happen due to line 403, will be at least n/a

Reply:
Good point. I'll pick that up next time.